src/Middleware/PaginationParser.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Choredo\Middleware;
+
+use Assert\Assertion;
+use Choredo\Pageable;
+use Choredo\PaginationCriteria;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use const Choredo\REQUEST_HANDLER_CLASS;
+
+class PaginationParser
+{
+    /**
+     * @var int
+     */
+    private $defaultLimit;
+    /**
+     * @var int
+     */
+    private $defaultOffset;
+    /**
+     * @var int
+     */
+    private $maxLimit;
+
+    /**
+     * PaginationParser constructor.
+     *
+     * @param int $defaultLimit
+     * @param int $defaultOffset
+     * @param int $maxLimit
+     */
+    public function __construct(int $defaultLimit = 10, int $defaultOffset = 0, int $maxLimit = 100)
```

Comment:
I would make these functions on the handler, rather than injecting them application wide.

Reply:
Yeah, definitely!